### PR TITLE
Add properties.schema to make authoring tool compatible

### PR DESCRIPTION
**Pull request by samumist**
Add properties schema to make this extension available for the Authoring tool.

### CODE REVIEW

**Comment by moloko** [comment]
@samumist fantastic, thanks for this. I'm assuming you've tested in the AT and you're happy it's all working?

File: properties.schema [hunk 1]
```diff
@@ -0,0 +1,188 @@
...
+              "legend": "Article Reveal",
+              "properties":{
+                "_isEnabled": {
+                    "type": "boolean",
+                    "required": false,
+                    "default": false,
+                    "title": "Article Reveal Enable",
+                    "inputType": {"type": "Boolean", "options": [true, false]},
+                    "validators": [],
+                    "help": "Enable the article reveal extenstion for this article."
```
Comment (moloko) [inline properties.schema]: typo: 'extenstion' should be 'extension'

Comment (samumist) [inline properties.schema]: Thanks for pointing out this, typo corrected.

**Comment by samumist** [comment]
Yes. I've tested in the AT and it works fine.

**Comment by tomgreenfield** [comment]
Resolves #10.

**Pull request merged**